docs_src/microservices/general/messagebus.md

```diff
@@ -211,6 +211,23 @@ The MessageBus configuration is in common configuration where the following chan
 !!! note
     The optional settings that apply to MQTT are already in the common configuration, so are not included above.
 
+#### Encode Message Envelope with CBOR
+
+This feature further reduces the payload size of all internal MQTT message envelopes, including system events, events, readings, and more.
+
+To enable CBOR encoding for the entire message envelope, set the environment variable `EDGEX_MSG_CBOR_ENCODE` for all EdgeX services. This can be done by adding the variable to the `.env` file before generating the docker-compose file. For guidance on generating the Docker Compose file, refer to the [Getting Started using Docker][1] guide.

```

Comment:
```suggestion
To enable CBOR encoding for the entire message envelope, set the environment variable `EDGEX_MSG_CBOR_ENCODE` to `true` for all EdgeX services. This can be done by adding the variable to the `.env` file before generating the docker-compose file. For guidance on generating the Docker Compose file, refer to the [Getting Started using Docker][1] guide.

```

Reply:
Updated.